### ISSUE TITLE
Undefined control sequence when used with citeproc

### ISSUE THREAD

**Issue by SandyJackson** [Wandmalfarbe/pandoc-latex-template#357]
I have suddenly started experiencing an issue with this template when running pandoc using citeproc to format citations. I was using it previously without issue (great template by the way, looks really good so thank you!) but suddenly today I start getting the following error:

```
! Undefined control sequence.
<argument> ...size \sf@size \selectfont \citeproc 
                                                  {ref-ref1}{1}, \ci...
l.581 \citeproc{ref-ref2}{2}}
```

The document will render to PDF fine without using the template (with citeproc and references working fine). It will also render fine if I don't attempt to use citeproc to manage the references.

I have reinstalled both latex and pandoc (v3.1.7). I've done every troubleshooting step I can think of but can't quite seem to get it to work.

**Comment by cagix** [comment]
In Pandoc 3.1.6 and 3.1.7 there were some issues and changes regarding the latex template in connection with citeproc. Presumably the Eisvogel template needs to be rebased on the current Pandoc latex template.

**Comment by SandyJackson** [comment]
Yup good advice. Simple fix. Thanks.

The tracked changes can be found here for anyone looking

https://github.com/jgm/pandoc-templates/commit/8a5b381c156c974b485197208c6dc86fce94b5c3

**Comment by cagix** [comment]
@Wandmalfarbe Could you please update the Eisvogel template?

**Comment by Wandmalfarbe** [comment]
@SandyJackson I just released a [new version 2.4.1](https://github.com/Wandmalfarbe/pandoc-latex-template/releases) that should fix the issue.
@cagix Thanks for the help!

**Comment by cagix** [comment]
de nada! 

fyi: it seems that there will be some more changes to the latex template soon (cf. https://github.com/jgm/pandoc/pull/9058) ...
